src/test/ui/union/union-sized-field.stderr

```diff
@@ -1,27 +1,27 @@
 error[E0277]: the trait bound `T: std::marker::Sized` is not satisfied
-  --> $DIR/union-sized-field.rs:14:5
+  --> $DIR/union-sized-field.rs:15:5
    |
-14 |     value: T, //~ ERROR the trait bound `T: std::marker::Sized` is not satisfied
-   |     ^^^^^^^^ `T` does not have a constant size known at compile-time
+15 |     unsized_field: T, //~ the trait bound `T: std::marker::Sized` is not satisfied
+   |     ^^^^^^^^^^^^^^^^ `T` does not have a constant size known at compile-time
    |
    = help: the trait `std::marker::Sized` is not implemented for `T`
    = help: consider adding a `where T: std::marker::Sized` bound
-   = note: no field of a union may have a dynamically sized type
+   = note: only the last field of a union may have a dynamically sized type
```

Comment:
This doesn't seem right. `union` doesn't have ordered fields. Maybe require that *at most* one field is unsized?

Reply:
It's not really right, I just did it this way for now to avoid having to refactor things. I can make it more general if I must :)